src/style.css

```diff
@@ -1 +1,86 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  position: relative;
+}
+
+.search__input {
+  position: relative;
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 20px;
+  background-position: 26px;
+  padding-left: 62px;
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-size: 19px;
+  font-size: 16px;
+}
+
+.search-bar::before {
+  content: '';
+  position: absolute;
+  background: url('/src/images/Search.svg') no-repeat;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+}
+
+.search-bar--small::before {
+  background-size: cover;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search__input:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  color: #3d4e61;
+  font-weight: 800;
+}
+
+.search__input--big {
+  height: 70px;
+}
+
+.search__input--small {
+  height: 42px;
+  background-position: 13px;
```

Comment:
The 'background-position' for '.search__input--small' should also include the vertical position for clarity, even if it's the default value.